hack/ci/create_devstack.sh

```diff
@@ -3,7 +3,7 @@
 # Copyright 2021 The Kubernetes Authors.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
+# may not use this file except in compliance with the License.
```

Comment:
I think the CI got upset that you dropped this

Reply:
I submit this PR when due to not drink my tea that day~ :)